### PR TITLE
Compose buildFeature flag could be removed after AGP 8.5.1

### PR DESCRIPTION
**Pull request by Goooler**
It will be enabled after `org.jetbrains.kotlin.plugin.compose` is applied.

### CODE REVIEW

**Comment by JunkFood02** [comment]
Oops! The ongoing development of seal is in the v2 branch, where I've already migrated to [Kotlin 2.0](https://github.com/JunkFood02/Seal/commit/78644b6ab17056ee37f70790692f8c173f58e317). I just forgot to update the contributing docs. Thanks for bringing this up!

File: app/build.gradle.kts [hunk 1]
```diff
@@ -161,10 +162,6 @@ android {
         }
     }
 
-    buildFeatures {
-        compose = true
-    }
```
Comment (Goooler) [inline app/build.gradle.kts:166]: 你那边可以重新打开这个 PR 吗，想把这个改动推上去，然后 rebase 一下分支就可以了

**Comment by JunkFood02** [comment]
thx! 

> Compose buildFeature flag could be removed after AGP 8.5.1

Do you have the changelog or a blog post refering to this?

**Comment by Goooler** [comment]
之前在推上面看到的 Compose 组的人说的，找不到出处了。这个标记位是控制 Compose 预览的，之前也可以关掉，关掉就没法看到预览，现在会默认开启。

**Pull request merged**